### PR TITLE
feat: Polysub addition

### LINKED ISSUE #26557: polysub

### PR DESCRIPTION
**Pull request by samuelmaina**
# PR Description

## Related Issue 

Close #26557

## Checklist

- [x] Did you add a function?
- [x] Did you add the tests?
- [ ] Did you run your tests and are your tests passing?
- [x] Did pre-commit not fail on any check?
- [x] Did you follow the steps we provided?

### CODE REVIEW

**Review by samuelmaina — COMMENTED** [review]
The change is good.

**Comment by samuelmaina** [comment]
Hello @DragosStoican , 
I am not able to run the test.  Whenever I run the tests, I am getting this error 
```
ERROR: not found: /Users/sam/ivy/ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_creation.py::test_polysub
(no name '/Users/sam/ivy/ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_creation.py::test_polysub' in any of [<Module test_creation.py>])
```
I also did not modify the `ivy_tests/array_api_testing/write_array_api_tests_k_flag.py` and I don't know why the `array-api-intelligent-tests-pr / run_tests (pull_request)`worflow  is failing

File: ivy/functional/backends/paddle/experimental/creation.py [hunk 1]
```diff
@@ -223,3 +223,32 @@ def mel_weight_matrix(
         upper_edge_hertz,
     )
     return paddle.transpose(mel_mat, (1, 0))
+
+
+def polysub(poly1_coeffs: paddle.Tensor, poly2_coeffs: paddle.Tensor) -> paddle.Tensor:
+    """
+    Calculate the difference of two polynomials.
```
Comment (DragosStoican) [inline ivy/functional/backends/paddle/experimental/creation.py:230]: There is no need for docstrings in our backend implementations. Can you please remove this?

**Comment by DragosStoican** [comment]
Hi @samuelmaina . I ran the tests for you and got the following error:
`file /opt/project/ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_creation.py, line 409
...
E       fixture 'dtype' not found`

It's due to the fact that you have not defined the dtype parameter in the `handle_test` decorator, and also `array` is not a valid keyword.

I cannot tell you for sure why your tests aren't working, but I'd recommend you to take a closer look at our setup guide if you haven't already [setting up](https://unify.ai/docs/ivy/overview/contributing/setting_up.html), or of course reaching out on discord with more details so we can help you!

Other than that I think this is close to merging, just iron out the errors in the test and hopefully you get them all passing soon. Thanks for your contribution!

File: ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_creation.py [hunk 2]
```diff
@@ -406,6 +406,29 @@ def test_ndindex(dtype_x_shape):
         assert index1 == index2
 
 
+@handle_test(
+    fn_tree="functional.ivy.experimental.polysub",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("array"),
```
Comment (DragosStoican) [inline ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_creation.py:412]: array is not a valid keyword, you should use valid

**Review by samuelmaina — COMMENTED** [review]
Comments cleanup fine

**Comment by samuelmaina** [comment]
Hello @DragosStoican , I have even deleted the miniconda version and installed it anew and  followed the installation steps in the guide and I am still getting the same error. I have asked the issue at 
 https://discord.com/channels/799879767196958751/982738436383445073/1159897999430930452 and https://discord.com/channels/799879767196958751/933380219832762430/1159019380152209418.
I am running any test using run button in pycharm or `python -m pytest ivy_tests/test_ivy/test_functional/test_core/test_creation.py::test_zeros_like --no-header --no-summary -q`  in the terminal
 However, I have corrected the issues that you mentioned above. Can you see if the tests run on your end kindly.

File: ivy/functional/ivy/experimental/creation.py [hunk 3]
```diff
@@ -33,9 +33,8 @@ def vorbis_window(
     dtype: Optional[Union[ivy.Dtype, ivy.NativeDtype]] = None,
     out: Optional[ivy.Array] = None,
 ) -> ivy.Array:
-    """
-    Return an array that contains a vorbis power complementary window of size
-    window_length.
+    """Return an array that contains a vorbis power complementary window of
+    size window_length.
```
Comment (DragosStoican) [inline ivy/functional/ivy/experimental/creation.py:37]: Make sure you are not modifying any existing code in the file. It looks like you've reformatted a lot of the docstrigs, could you revert that change please?

File: ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_creation.py [hunk 4]
```diff
@@ -406,6 +406,29 @@ def test_ndindex(dtype_x_shape):
...
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
+        max_num_dims=1,
+        min_value=-1000,
+        max_value=1000,
+    ),
+    test_with_out=st.just(True),
+    test_instance_method=st.just(False),
+)
+def test_polysub(*, dtype_and_x, dtype, test_flags, backend_fw, fn_name, on_device):
```
Comment (DragosStoican) [inline ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_creation.py:420]: `dtype` is not defined in `handle test`, which is why your test is currently failing.

**Pull request closed without merging**